tox.ini

```diff
@@ -0,0 +1,50 @@
+[tox]
+envlist =
+    coverage
+    flake8
+recreate = True
+
+[testenv]
+basepython = python3
+setenv =
+    FAKE_TRANSLATE = always
+    PYTHONPATH = {toxinidir}/fake_deps
+
+deps =
+    urwid==1.2.1
+    PyYAML
+    attrs
+    jsonschema
+    pyudev
+    requests
+    requests-unixsocket
+
+[testenv:flake8]
+deps =
+    flake8==3.5.0
+    pycodestyle==2.3.1
+    pyflakes==1.6.0
+
+commands = {envpython} -m flake8 \
+    {posargs:subiquity/ subiquitycore/ console_conf/ bin/}
+
+[testenv:py3]
+deps =
```

Comment:
is there any reason to even have the py3 env if we are going to run coverage by default?

Reply:
its a lot faster. thats the big thing.  *I'll* use it.

I use 'tox-venv'.  it doesnt read the 'setenv' yet, so i have to do it  myself.
but...
if i'm not looking at coverage, thats almost 3 seconds of my life i get back.
if we wanted to drop it at some point, then i'd be fine.

$ time FAKE_TRANSLATE=always PYTHONPATH=$PWD/fake_deps:. tox-venv py3 - >/dev/null 2>&1

real	0m1.898s
user	0m1.756s
sys	0m0.086s

$ time FAKE_TRANSLATE=always PYTHONPATH=$PWD/fake_deps:. tox-venv coverage - >/dev/null 2>&1

real	0m4.536s
user	0m4.378s
sys	0m0.136s